README.md

```diff
@@ -1306,6 +1306,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Udokaku Ugochukwu](https://github.com/UdokaVrede)
   - [Ujjwal Gupta](https://github.com/UG-SEP)
   - [Ujjwal Gupta](https://github.com/ujjwalgupta91250)
+  - [Ugochukwu Benjamin](https://github.com/Benji918)
```

Comment:
Please add your name in line 1307, after Udokaku Ugochukwu.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable